frontend/src/components/MeetingRoom/ToolbarOverflowMenu/ToolbarOverflowMenu.tsx

```diff
@@ -77,42 +77,42 @@ const ToolbarOverflowMenu = ({
               bottom: '80px',
             }}
           >
-            {shownButtons <= 1 && (
+            {shownButtons < 1 && (
               <ScreenSharingButton
                 toggleScreenShare={toggleShareScreen}
                 isSharingScreen={isSharingScreen}
                 isViewingScreenShare={isViewingScreenShare}
                 isOverflowButton
               />
             )}
-            {shownButtons <= 2 && (
+            {shownButtons < 2 && (
               <LayoutButton isScreenSharePresent={isViewingScreenShare} isOverflowButton />
             )}
-            {shownButtons <= 3 && (
+            {shownButtons < 3 && (
```

Comment:
any particular reason to switch from `<=` to `<` at this point? 🤔

Reply:
Good question! It fixes the buggy buttons Sachin noticed. If they're both using `=`, they are duplicated in the overflow menu and the toolbar.